main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -70,8 +70,8 @@ sealed trait Completion {
         textEdit = textEdit,
         insertTextFormat = insertTextFormat,
         kind = CompletionItemKind.Enum)
-    case Completion.TypeEnumCompletion(enumSym, nameSuffix, priority, textEdit, documentation) =>
-      CompletionItem(label = s"${enumSym.name}$nameSuffix",
+    case Completion.EnumCompletion(enumSym, nameSuffix, priority, textEdit, documentation) =>
```

Comment:
What is `nameSuffix`? I don't think we should need that.

Reply:
No clue honestly. It was just part of the original structure when creating a `CompletionItem` for enums.